projects/cashmere/src/lib/pipes/null-or-empty/null-or-empty.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    'name': 'nullOrEmpty'
+})
+export class NullOrEmptyPipe implements PipeTransform {
+
+    public transform(value: string, altText: string): string {
+        if (value) {
+            value = value.trim();
```

Comment:
What if an object is passed? What if the value is falsey like `0` or `false` or `NaN` but not `null` or `''`?

Reply:
I would also add test cases for all these cases.

Reply:
perhaps you could do something like this:

```ts
if (value) {
  value = value.toString().trim();
}
```